views/js/controller/Delivery/monitoring.js

```diff
@@ -119,40 +121,79 @@ define([
+                                    messageContext += '<br>' + encode.html(response.error);
+                                }
+                            }
+
+                            feedback().error(__('Something went wrong ...') + '<br>' + messageContext, {encodeHtml: false});
                         }
                     });
                 }
             }
 
             // request the server to authorise the selected delivery executions
             function authorise(selection) {
-                execBulkAction('authorize', __('Authorize Session'), selection, function(selection, reason){
-                    request(authoriseUrl, selection, reason, __('Sessions authorized'));
+                execBulkAction('authorize', __('Authorize Session'), selection, function(sel, reason){
+                    request(authoriseUrl, sel, reason, __('Sessions authorized'));
                 });
             }
 
             // request the server to pause the selected delivery executions
             function pause(selection) {
-                execBulkAction('pause', __('Pause Session'), selection, function(selection, reason){
-                    request(pauseUrl, selection, reason, __('Sessions paused'));
+                execBulkAction('pause', __('Pause Session'), selection, function(sel, reason){
+                    request(pauseUrl, sel, reason, __('Sessions paused'));
                 });
             }
 
             // request the server to terminate the selected delivery executions
             function terminate(selection) {
-                execBulkAction('terminate', __('Terminate Session'), selection, function(selection, reason){
-                    request(terminateUrl, selection, reason, __('Sessions terminated'));
+                execBulkAction('terminate', __('Terminate Session'), selection, function(sel, reason){
+                    request(terminateUrl, sel, reason, __('Sessions terminated'));
                 });
             }
-            
+
             // report irregularities on the selected delivery executions
             function report(selection) {
-                execBulkAction( 'report', __('Report Irregularity'), selection, function(selection, reason){
-                    request(reportUrl, selection, reason, __('Sessions reported'));
+                execBulkAction( 'report', __('Report Irregularity'), selection, function(sel, reason){
+                    request(reportUrl, sel, reason, __('Sessions reported'));
                 });
             }
-            
+
+            // display the session history
+            function showHistory(selection) {
+                var urlParams = {
+                    testCenter : testCenterId,
+                    session: selection
+                };
+                if (deliveryId) {
+                    urlParams.delivery = deliveryId;
+                }
+                location.href = helpers._url('sessionHistory', 'Reporting', 'taoProctoring', urlParams);
```

Comment:
is it the `location` from `window.location` ? If yes, please specify it (so everybody knows where this is from).